### PR TITLE
Issue: #240 fixed (BoundingBoxes3D don't render correctly with padded images)

### PR DESCRIPTION
**Pull request by FlorianCoissac**
To try it out I used a code like this: (try it out with an actual image to check it out)

import numpy as np
import torch
from aloscene import Frame, Labels, BoundingBoxes3D, CameraExtrinsic, CameraIntrinsic, BoundingBoxes2D

frame = Frame(np.zeros((3, 500, 500)), normalization="01")
# frame = Frame("/home/flo/rosbags/vb_office_cartons/RGB/2736_color.png", normalization="01")
frame.append_cam_intrinsic(CameraIntrinsic(focal_length=616.0, plane_size=frame.HW))
frame.append_cam_extrinsic(CameraExtrinsic(torch.eye(4, dtype=torch.float32)))
label = Labels([1], names=["name"])

box3d = BoundingBoxes3D([[0.75, 0.05, 2.5, 0.4, 0.5, 0.25, 0.5]], labels=label)
frame.append_boxes3d(box3d)

box2d = BoundingBoxes2D(
    [[200, 200, 200, 300]], boxes_format="xcyc", absolute=True, frame_size=(500, 500), labels=label
)
frame.append_boxes2d(box2d)

# frame1 = Frame("/home/flo/Pictures/continuous_walss.png", normalization="01")

# frames = torch.cat([frame.batch(), frame1.batch()], dim=0)
frame1 = frame.pad((0.3, 0.1), (0.1, 0.3))

print(frame.size(), frame1.size())
frame.get_view().render()
frame1.get_view().render()
# frames.get_view().render()

### CODE REVIEW

**Comment by Data-Iab** [comment]
-- > alobugdays
